src/main/java/org/spongepowered/mod/mixin/core/entity/player/MixinEntityPlayerMP.java

```diff
@@ -111,6 +118,14 @@ public MixinEntityPlayerMP(World worldIn, com.mojang.authlib.GameProfile gamepro
         return new SpongeMessageText.SpongeMessageTextBuilder(getGameProfile().getName()).build();
     }
 
+    public boolean playermp$getAllowFlight() {
```

Comment:
`getAllowFlight` and `setAllowFlight` are already implemented in the above `MixinEntityPlayer`, no need to redo the methods again.